### PR TITLE
Draft - Resources and Configurations

### PR DESCRIPTION
**Pull request by cmpadden**
- Standardizes asset definitions with _automations_ presentation (eg. including table metadata)
- Fixes schedule definition `run_config` with `to_config_dict`

### CODE REVIEW

**Comment by cmpadden** [comment]
Example of how table of presentations will render:
image

File: project-blitz/project_blitz/__init__.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 from project_blitz import automation, resources_and_configurations
 
 
-defs = automation.defs
-# defs = resources_and_configurations.defs
+# defs = automation.defs
```
Comment (cmpadden) [inline project-blitz/project_blitz/__init__.py:4]: @PedramNavid / @tacastillo how do we want to manage project definitions? Right now I just toggle between them with comments.

Comment (PedramNavid) [reply project-blitz/project_blitz/__init__.py:4]: we have a couple options, but i think maybe the simplest is to have one module per talk, and the definitions in that module, with no global definitions.
then you can run `dagster dev -m my_module`

Comment (cmpadden) [reply project-blitz/project_blitz/__init__.py:4]: Confirmed that the following works great - thank you.
```
dagster dev -m project_blitz.automation
# or
dagster dev -m project_blitz.resources_and_configurations
```

Comment (cmpadden) [reply project-blitz/project_blitz/__init__.py]: Added an option to choose the project when using `make run`:
```
> make run

Select the example project to run:
1) automation
2) resources-and-configuration
#? 2
```

**Pull request merged**